cpp/velox/shuffle/VeloxShuffleReader.cc

```diff
@@ -286,9 +286,9 @@ RowVectorPtr readRowVectorInternal(
     memory::MemoryPool* pool) {
   auto header = readColumnBuffer(batch, 0);
   uint32_t length;
-  mempcpy(&length, header->data(), sizeof(uint32_t));
+  memcpy(&length, header->data(), sizeof(uint32_t));
```

Comment:
Any particular reason for this?

Reply:
`mempcpy` is a gnu extension function, apple clang does not support that